### PR TITLE
Attach original AJAX framework events to Snowboard in Backend

### LINKED ISSUE #1233: Framework extras missing in backend
### Winter CMS Build

dev-develop

### PHP Version

8.1

### Database engine

MySQL/MariaDB

### Plugins installed

_No response_

### Issue description

Trying to use a button in a backend view with a `data-attach-loading` does not work. 

Note: this works fine on a frontend page

### Steps to replicate

Add the following button to a backend controller:
```html
<button data-attach-loading data-request=onAjax>TEST</button>
```

### Workaround

_No response_

### PR DESCRIPTION
**Pull request by bennothommo**
Replaces #1234, fixes #1233.

Instead of loading `framework.extras.css`, which appears to be excluded from the Backend, this PR hooks the necessary original AJAX framework events into Snowboard's events, allowing the applicable events to execute UI functionality such as `data-attach-loading` and the Stripe loader.

This is technically a stop-gap solution for now, since it doesn't emulate a full Snowboard request, so there's the potential for custom Snowboard event listeners to break if they are relying on Snowboard request information, but I'm not aware of that happening yet in core.

### CODE REVIEW

**Comment by mjauvin** [comment]
Nice!

File: modules/backend/assets/ui/js/ajax/Handler.js [hunk 1]
```diff
@@ -46,6 +50,42 @@ export default class Handler extends Snowboard.Singleton {
...
+                options: {},
+            });
+        });
+        document.addEventListener('$ajaxFail', (event) => {
+            this.requests[event.target].reject(event.detail.textStatus);
+            this.snowboard.globalEvent('ajaxDone', event.detail.data, {
+                element: event.target,
+                options: {},
+            });
+        });
```
Comment (LukeTowers) [inline modules/backend/assets/ui/js/ajax/Handler.js:92]: Can you add this functionality to the functions comment at the top of the class?

Comment (LukeTowers) [reply modules/backend/assets/ui/js/ajax/Handler.js:92]: @bennothommo ping

Comment (bennothommo) [reply modules/backend/assets/ui/js/ajax/Handler.js:92]: Cheers for the ping @LukeTowers. I've added it

**Pull request merged**